schema/google/showcase/v1beta1/sequence.proto

```diff
@@ -166,7 +166,6 @@ message StreamingSequenceReport {
     // The status returned to the attempt.
     google.rpc.Status status = 5;
 
-    string content_sent = 6;
```

Comment:
Note: Since we discussed it offline, _this_ change will precipitate a regeneration PR - that is OK, that is intentional.

Reply:
Yea thanks for clarifying!